guardian/newsletters-nx
# Allow skipping to section in data collection wizard

## What does this change?

- Allows skipping between steps. Requires that a  name be added as subsequent steps rely on a name being set.
- Removes the braze and identiy steos from the data collection wizard. These steps will be re-added in the lanch step in a later PR

## How to test

Chack out the brach & ensure it is possible to add update the steps out of sequence.


## Have we considered potential risks?

Always

## Images

![hotStepper](https://user-images.githubusercontent.com/3277259/233335844-fea2b922-7b2d-4313-818d-a2fcecc1fa8a.gif)
